dev/breeze/setup.cfg

```diff
@@ -66,6 +67,7 @@ where=src
 console_scripts=
     Breeze2=airflow_breeze.breeze:main
     airflow-freespace=airflow_ci.freespace:main
+    test-examples-of-prod-image-building=airflow_ci.test_examples_of_prod_image_building:main
```

Comment:
I think we do not need that really

In our case simply `python -m pytest <pytes parameters here> ./docker_tests/test_examples_of_prod_image_building.py` should do the job nicely.

Reply:
I see Jarek, I thought I had to apply the Entrypoint idea (like free-space), and move the files to airflow_ci, as many as possible. Is it due to complexity?

Reply:
Yeah. It's not needed really, because we only run the tests as "pytest" tests. In case of the "airflow-freespace" that was a "standalone" python code that we had to turn into entrypoing - but in this case we really want to just run pytest tests and `pytest` is an entrypoint on its own. 

So our "entrypoint" is "pytest" and our "test_*.py" is simply parameter passed to pytest to tell it what it shoudl run. The `python -m pytest` is just recommended way of running `pytest` entrypoint: https://docs.pytest.org/en/6.2.x/usage.html#calling-pytest-through-python-m-pytest - we could also do it via `pytest` (and there it will be similar to `airlfow-freespace` but i prefer to use `python -m` wherever possible as it is more portable.